oshdb/src/main/java/org/heigit/ohsome/oshdb/OSHDB.java

```diff
@@ -8,6 +8,26 @@
   public static final long GEOM_PRECISION_TO_LONG = 10000000L;
   public static final double GEOM_PRECISION = 1.0 / GEOM_PRECISION_TO_LONG;
 
+  /**
+   * Convert a double precision coordinate to fix precision long value
```

Comment:
Missing space after javadoc comment: https://jenkins.ohsome.org/job/oshdb/view/change-requests/job/PR-320/2/checkstyle/fileName.-449784562/type.-510536328/source.e4b60da5-07ad-4f97-b28c-1e166328b54c/#11 (same below)

Reply:
is there a typo? “_…to fix**ed** precision long value_” would be better?! (same below also)

Reply:
```suggestion
   * Converts a double precision coordinate to a fixed precision long value.
```